packages/uniwind/src/metro/compileVirtual.ts

```diff
@@ -9,18 +10,27 @@ import { serializeJSObject } from './utils'
 type CompileVirtualConfig = {
     cssPath: string
     css: string
-    candidates: Array<string>
     platform: Platform
     themes: Array<string>
     polyfills: Polyfills | undefined
 }
 
-export const compileVirtual = async ({ candidates, css, cssPath, platform, themes, polyfills }: CompileVirtualConfig) => {
+export const compileVirtual = async ({ css, cssPath, platform, themes, polyfills }: CompileVirtualConfig) => {
     const compiler = await compile(css, {
         base: path.dirname(cssPath),
         onDependency: () => void 0,
     })
-    const tailwindCSS = compiler.build(candidates)
+    const scanner = new Scanner({
+        sources: [
+            ...compiler.sources,
+            {
+                negated: false,
+                pattern: '**/*',
+                base: path.dirname(cssPath),
```

Comment:
@Brentlok I believe this introduced a bug when the CSS input file is not in the root or src directory tree, such as:

```
app
   - ....
styles
   - global.css
``` 

Metro config:

```js
const uniwindConfig = withUniwindConfig(config, {
  cssEntryFile: "./styles/global.css",
});

module.exports = uniwindConfig;
```

Based on my brief debugging with limited context, changing this to `process.cwd()` fixes the issue 🙈

Reply:
Placing the CSS input file in the root directory significantly increases build time, likely due to scanning the `node_modules` and hidden folders 😬